gen/dcos-config.yaml

```diff
@@ -1125,3 +1117,100 @@ package:
     content: {{ fault_domain_detect_contents }}
 {% case "false" %}
 {% endswitch %}
+  - path: /etc/telegraf.env
+    content: |
+      TELEGRAF_CONFIG_FILE=/opt/mesosphere/etc/telegraf/telegraf.conf
+      TELEGRAF_CONFIG_DIR=/opt/mesosphere/etc/telegraf/telegraf.d/
+  - path: /etc/telegraf/telegraf.conf
+    content: |
+      # Telegraf config for all nodes
+      [global_tags]
+        # Tags to be applied to all metrics.
+        dcos_cluster_name="{{ cluster_name }}"
+        dcos_cluster_id="$DCOS_CLUSTER_ID"
```

Comment:
Does telegraf understand environment variables like this in its config?

Reply:
Yeah, it's nice. https://docs.influxdata.com/telegraf/v1.7/administration/configuration/#environment-variables